src/app/login/login-model.ts

```diff
@@ -0,0 +1,9 @@
+export class LoginModel {
+
+    constructor(
+
+        public username: string,
+        public password: string
+    ) { }
+
+}
```

Comment:
It's better to have interface instead of class as we don't have any methods for this model, so we can use it as a type directly instead of instantiating it.